CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 * [#443](https://github.com/clojure-emacs/clojure-mode/issues/443): Fix behavior of `clojure-forward-logical-sexp` and `clojure-backward-logical-sexp` with conditional macros.
 * [#429](https://github.com/clojure-emacs/clojure-mode/issues/429): Fix a bug causing last occurrence of expression sometimes is not replaced when using `move-to-let`.
 * [#423](https://github.com/clojure-emacs/clojure-mode/issues/423): Make `clojure-match-next-def` more robust against zero-arity def-like forms.
+* Fix namespace font-locking
```

Comment:
You can also mention here the GH issue you've fixed. This sentence should end with a `.`.

Reply:
I did the fix out of pure excitement of having found "a pristine spot on a wall I can spray my tag on". Ok seriously I there is no GH issue for my PR. Although the #280 seems to be closely related.